internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -127,6 +128,11 @@ func (opts *Opts) initStore(ctx context.Context) func() error {
 
 func (opts *Opts) PreRun(ctx context.Context, outWriter io.Writer) error {
 	opts.setTier()
+
+	if opts.CurrentIP && len(opts.IPAddresses) > 0 {
+		return fmt.Errorf("cannot use %s and %s, please use only one of the flags", flag.CurrentIP, flag.AccessListIP)
+	}
```

Comment:
as a rule of thump we use the WebUI as guidance on behaviour if the WebUI let's you add multiple in a single step we should do it otherwise this suggestion is a bit of an edge-case for an advanced user that may want their current IP but also some external systems